tmt/utils.py

```diff
@@ -51,7 +51,7 @@
 log = fmf.utils.Logging('tmt').logger
 
 # Default workdir root and max
-WORKDIR_ROOT = '/var/tmp/tmt'
+WORKDIR_ROOT = os.getenv('TMT_RUN_DIR', '/var/tmp/tmt')
```

Comment:
IMO we should have this as a default value when envar is missing and check TMT_RUN_DIR when WORKDIR_ROOT creation is requested. Not during the import.

Reply:
Makes sense to me.

Reply:
agreed